specification/batch/resource-manager/Microsoft.Batch/stable/2021-06-01/BatchManagement.json

```diff
@@ -4272,6 +4267,11 @@
           "type": "string",
           "title": "The password to log into the registry server."
         },
+        "registryServer": {
+          "type": "string",
+          "title": "The registry URL.",
+          "description": "If omitted, the default is \"docker.io\"."
+        },
         "identityReference": {
```

Comment:
Do you have a PR# which merge 2021-06-01 version? 
Not sure whether it is better to put it in last one after identityReference if registerServer is new added.

Reply:
Here's 2021-06-01's PR: https://github.com/Azure/azure-rest-api-specs/pull/14862

registryServer is just being moved. IdentityReference was added in 2021-06-01.